content/en/api/synthetics/post_test.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Create a test
+type: apicontent
+order: 24.1
+external_redirect: /api/#create-tests
+---
+
+## Create a test
+
+Create a Synthetics test to initiate and configure the tests you want Datadog to send to your API endpoints or to your browser app. You can configure the endpoints being tested, the number of tests, and where they are coming from.
+
+##### Arguments
+
+*   **`assertions`** - _required_ - The function where you define when you want Datadog to alert you. Assertions are defined by: `operator`, `property`, `type`, and `target`.
+*   **`operator`** - _required_ - The action that you want tested. Valid operators depend on the property `type` being tested.
```

Comment:
Would rephrase to: `operator`: Defines how to compare the `target` and the actual value from the response. The list of possible operators depends on the type of assertions... + list of the valid operators per type.